lib/spaceship/launcher.rb

```diff
@@ -88,6 +88,9 @@ def certificate
 
     # @return (Class) Access the provisioning profiles for this spaceship
     def provisioning_profile
+      app
+      device
+      certificate
       Spaceship::ProvisioningProfile.set_client(@client)
```

Comment:
Wouldn't we need to solve the same profile for all the other methods, like `certificate`?